pkgs/top-level/python-packages.nix

```diff
@@ -8105,6 +8105,10 @@ in {
     withWebKit = true;
   };
 
+  pyqt6 = callPackage ../development/python-modules/pyqt/6.x.nix { };
```

Comment:
```suggestion
  pyqt6 = callPackage ../development/python-modules/pyqt/6.nix { };
```

Reply:
The existing 5 package in that directory is 5.x, followed the same scheme.

Reply:
At least in the python-modules directory multiple versions always have the least amount digest required to identify the file.